src/app/components/NotificationForm/NotificationFormContainer/index.tsx

```diff
@@ -0,0 +1,240 @@
+/**
+ *
+ * NotificationForm
+ *
+ */
+
+import React, { useReducer, useState, useEffect } from 'react';
+import axios from 'axios';
+import { useAccount } from '../../../hooks/useAccount';
+import { NotificationFormComponent } from '../NotificationFormComponent';
+import { Sovryn } from '../../../../utils/sovryn';
+
+export function NotificationForm() {
+  const mailApiKey = process.env.REACT_APP_MAIL_API_KEY;
+  const mailSrv = process.env.REACT_APP_MAIL_SRV;
```

Comment:
Api keys should not be in browser side, we need them on server side. And maybe new project for this kind thing if we dont already.

```suggestion
```